tests/unish/UnishIntegrationTestCase.php

```diff
@@ -99,6 +104,13 @@ public function drush($command, array $args = [], array $options = [], $expected
         return $return;
     }
 
+    protected function setStdin($contents)
+    {
+        $path = $this->writeToTmpFile($contents);
```

Comment:
If you write to `getSandbox()`, then cleanDirs() will clean up automatically. Would remove all changes in UnishTestCase ... Just a thought.

Reply:
Yeah I like to run locally with UNISH_DIRTY, and I still want these to go away. Maybe segregate the sandbox into invariant-deletion and only-if-not-dirty deletion would work

Reply:
Sure, an alwaysDelete subdir of the sandbox sounds good